libobs/util/platform-cocoa.m

```diff
@@ -417,3 +420,89 @@ uint64_t os_get_proc_virtual_size(void)
 		return 0;
 	return taskinfo.virtual_size;
 }
+
+/* Obtains a copy of the contents of a CFString in specified encoding.
+ * Returns char* (must be bfree'd by caller) or NULL on failure.
+ */
+char* cfstr_copy_cstr(CFStringRef cfstring, CFStringEncoding cfstring_encoding)
+{
+	if (!cfstring) return NULL;
```

Comment:
Style:  Don't include multiple statements on a single line.  Place the `return` on its own line.